include/wx/generic/treectlg.h

```diff
@@ -243,8 +244,8 @@ class WXDLLIMPEXP_CORE wxGenericTreeCtrl : public wxTreeCtrlBase,
     unsigned short       m_indent;
     int                  m_lineHeight;
     wxPen                m_dottedPen;
-    wxBrush             *m_hilightBrush,
-                        *m_hilightUnfocusedBrush;
+    wxBrush              m_hilightBrush,
+                         m_hilightUnfocusedBrush;
```

Comment:
These members are protected, so user code can get at them. I didn't make this change for fear of breaking something, especially this close to making a new release.

Reply:
I didn't even think about this, to be honest, but I don't believe we guarantee compatibility for random, undocumented, protected members (and yes, of course they should have been private in the first place...). Unfortunately it's difficult to prove that nobody really uses them, but it seems like their only occurrences are in the various clones of this file in the other projects, not in the projects actually using these variables in the base class.